src/main/java/org/swisspush/redisques/handler/RedisquesHttpRequestHandler.java

```diff
@@ -138,7 +138,7 @@ private RedisquesHttpRequestHandler(
         this.exceptionFactory = exceptionFactory;
         this.queueStatsService = new QueueStatsService(
                 vertx, eventBus, redisquesAddress, queueStatisticsCollector, dequeueStatisticCollector,
-                exceptionFactory, queueStatsRequestQuota);
+                exceptionFactory, queueStatsRequestQuota,modConfig.isDequeueStatsEnabled());
```

Comment:
look for code format like spaces and indention

Reply:
ok